imutils/video/webcamvideostream.py

```diff
@@ -3,10 +3,12 @@
 import cv2
 
 class WebcamVideoStream:
-	def __init__(self, src=0, name="WebcamVideoStream"):
+	def __init__(self, src=0, name="WebcamVideoStream" resolution=(640, 480)):
```

Comment:
You missed a coma between _name="WebcamVideoStream"_ and _resolution=(640, 480)_ .
Otherwise everything is ok for me !